components/AddForm.js

```diff
@@ -0,0 +1,95 @@
+import styled, { css } from 'styled-components';
+import Buttons from './Buttons';
+import useStore from '../hooks/useStore';
+import { useState } from 'react';
+import { nanoid } from 'nanoid';
+
+export default function AddForm() {
+  const [inputName, setInputName] = useState('');
+  const [inputInformation, setInputInformation] = useState('');
+  const addCharacter = useStore(state => state.addCharacter);
+  const toggleToast = useStore(state => state.toggleToast);
+
+  function submitForm(event) {
```

Comment:
Suggestion (for later): instead of putting "check out your character under my characters" (or something) into your notification, you could include some routing to automatically direct the user toward my characters after submitting their character. You might need to figure out the timing (so that it happens after your notification has timed out), but other than that this might be beneficial. Although that's really up to you.

Reply:
I think for this later again :) Thanks!